### DISCUSSION TITLE
Mass Update Old Data With New One

### DISCUSSION THREAD

**Discussion by lucius100** [pocketbase/pocketbase#4895] [Q&A]
Does there any way like I can export my data into excel, then organize the old data  and mass update certain fields with new data ?
Or the only way is to update via api one by one ?

**Comment by thangngoc89** [comment]
I would say run SQL statement on the `data.db` directly.
You could use the default [SQLite cli](https://sqlite.org/cli.html) or plenty of other similar softwares.
